### PR TITLE
Major changes 2

### PR DESCRIPTION
**Pull request by gradedSystem**
## Major description changes:
- Changed the scripts from Python2 obsolete version to Python3
- Changed the links to the sources for example `territories.json` is currently moved to a new repo
- Added a new csvtojson function which replaces Makefile
- Updated the codes with some modifications like `scripts/edgar.py`

### CODE REVIEW

File: .github/workflow/actions.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+name: Run Makefile with Selenium
+
+on:
+  # Schedule to run every 2 minutes
+  schedule:
+    - cron: '*/2 * * * *'
```
Comment (anuveyatsu) [inline .github/workflow/actions.yml:6]: We want to run it on a monthly basis, eg, on the 1st day of each month.

File: .github/workflow/actions.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+  # Run on push to master branch
+  push:
+    branches:
+      - master
```
Comment (anuveyatsu) [inline .github/workflow/actions.yml:11]: update master to main branch

File: .github/workflow/actions.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  # Run on pull requests targeting the master branch
+  pull_request:
+    branches:
+      - master
```
Comment (anuveyatsu) [inline .github/workflow/actions.yml:16]: ditto

File: .github/workflow/actions.yml (hunk 1 above)
Comment (gradedSystem) [reply .github/workflow/actions.yml]: Fixed

File: .github/workflow/actions.yml (hunk 2 above)
Comment (gradedSystem) [reply .github/workflow/actions.yml]: Fixed

File: .github/workflow/actions.yml (hunk 3 above)
Comment (gradedSystem) [reply .github/workflow/actions.yml]: Fixed

File: .github/workflow/actions.yml [hunk 4]
```diff
@@ -0,0 +1,54 @@
+name: Run Makefile with Selenium
+
+on:
+  # Schedule to run every 2 minutes
```
Comment (anuveyatsu) [inline .github/workflow/actions.yml:4]: @gradedSystem this comment is now misleading...

**Pull request merged**